config/api_platform/resources.yaml

```
@@ -1,4 +1,64 @@
+          ['customer.mongodb.type.order', 'customer.mongodb.type.partial_search', 'mongodb.range']
+      ApiPlatform\Metadata\Get:
+      ApiPlatform\Metadata\Put:
+      ApiPlatform\Metadata\Patch:
+      ApiPlatform\Metadata\Delete:
+      ApiPlatform\Metadata\Post:
+
+  App\Customer\Domain\Entity\Customer:
+    paginationPartial: true,
+    paginationClientItemsPerPage: true
+    paginationViaCursor:
+      - { field: 'ulid', direction: 'DESC' }
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)